$OpenBSD: patch-configure,v 1.9 2013/08/11 14:04:06 ajacoutot Exp $

XXX patch evo_krb5_support.m4 to use krb5-config(1) and push upstream
(and remove the call to krb5-config from MODGNOME_CPPFLAGS)

--- configure.orig	Sun Aug 11 13:53:46 2013
+++ configure	Sun Aug 11 15:54:40 2013
@@ -15461,7 +15461,7 @@ $as_echo_n "checking for Kerberos 5... " >&6; }
 		esac
 
 		mitlibs="-lkrb5 -lk5crypto -lcom_err -lgssapi_krb5"
-		heimlibs="-lkrb5 -lcrypto -lasn1 -lcom_err -lroken -lgssapi"
+		heimlibs="`krb5-config --libs gssapi`"
 		sunlibs="-lkrb5 -lgss"
 		if ${ac_cv_lib_kerberos5+:} false; then :
   $as_echo_n "(cached) " >&6
